src/components/record/related-table-actions.tsx

```diff
@@ -519,29 +510,15 @@ const RelatedTableActions = ({
       setShowPureBinarySpinner(true);
 
       validateSessionBeforeMutation(() => {
-        const deleteResponse = (response: any) => {
-          setShowPureBinarySpinner(false);
-
-          // Show modal popup summarizing total # of deletions succeeded and failed
-          response.clickOkToDismiss = true;
-
-          // TODO: - improve partial success and use TRS to check delete rights before giving a checkbox
-          //       - some errors could have been because of row level security
-          dispatchError({
-            error: response,
-            isDismissible: true,
-            closeBtnCallback: () => {
-              // ask recordset to update the modal
-              if (!!containerRef.current) {
-                // NOTE: This feels very against React but the complexity of our flow control provider seems to warrant doing this
-                fireCustomEvent(CUSTOM_EVENTS.FORCE_UPDATE_RECORDSET, containerRef.current, {
-                  cause: LogReloadCauses.ENTITY_BATCH_UNLINK,
-                  pageStates: { updateResult: true, updateCount: true, updateFacets: true },
-                  response: response,
-                });
-              }
-            },
-          });
+        const deleteResponse = () => {
```

Comment:
This is no longer the response to the `deleteBatchAssociationTuples` request. This should be renamed or undo the change to "refactor" this function.